pkg/cmd/server/crypto/crypto.go

```diff
@@ -210,6 +216,7 @@ func GetServerCert(certFile, keyFile string, hostnames util.StringSet) (*TLSCert
 
 func (ca *CA) MakeServerCert(certFile, keyFile string, hostnames util.StringSet) (*TLSCertificateConfig, error) {
 	glog.V(2).Infof("Generating server certificate in %s, key in %s", certFile, keyFile)
+	fmt.Fprintf(ca.GetOut(), "Generating server certificate as %s, key as %s\n", certFile, keyFile)
```

Comment:
You were right, I was wrong... with just a little more complexity in the return values, I could keep the output out of the crypto lib.